### PR TITLE
Improve wizard logging

### LINKED ISSUE #9664: Use QLoggingCategory in new Wizard

### PR DESCRIPTION
**Pull request by fmoc**
Fixes #9664.

I'd like to extend the logging further, but this makes sense only after merging #9779.

### CODE REVIEW

File: src/gui/newwizard/logging.cpp [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#include "logging.h"
```
Comment (TheOneRing) [inline src/gui/newwizard/logging.cpp:1]: I guess we should have more than one category and thus we won't need the logging.h/cpp

Comment (TheOneRing) [reply src/gui/newwizard/logging.cpp:1]: As the category is only used in the .cpp then we won't need to make them available in the header with Q_DECLARE_LOGGING_CATEGORY

Comment (TheOneRing) [reply src/gui/newwizard/logging.cpp:1]: We could also look on https://api.kde.org/ecm/module/ECMQtDeclareLoggingCategory.html

Comment (fmoc) [reply src/gui/newwizard/logging.cpp:1]: We can use more specific categories.

**Review by TheOneRing — APPROVED** [review]

**Pull request merged**